### PR TITLE
Combat+

### PR DESCRIPTION
**Pull request by CollenN**
A massive update to the combat and damage system. Check the changelog for a list of newstuff.

Includes: blood trails, detective scanner tells you weight of item that
caused blood splatter, morphine, broken bones, body tracing, blood/vomit
drying, windedness, bleeding and more!

### CODE REVIEW

**Comment by SuperSayu** [comment]
Instead of updating atoms.dm and adminordrazine yourself you should rebase your code onto the latest version.

**Comment by Profakos** [comment]
Wait, no surgeries for broken bones? It won't add much to medbay if it is just "Stick 'em in cryo", I was hoping to see more scalpel action, even if it is just something that starts up natural healing of bones.  :(((

**Comment by Profakos** [comment]
Uh, 2-4% tick that cryo/clonax heals bones? That is terrible, I am making the surgery alternatives asap.

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -1218,5 +1238,21 @@
...
+			var/datum/limb/affecting = get_organ("chest")
+			if(prob(65))
+				apply_damage(1, BRUTE, affecting)
+		if("head" in broken)
+			if(prob(5) && stat == CONSCIOUS)
+				sleeping = 2
+				adjustBruteLoss(2)
+			if(prob(25))
+				adjustBrainLoss(1)
+		if((dna) && dna.mutantrace == "slime" || dna.mutantrace == "plant")
```
Comment (SuperSayu) [inline code/modules/mob/living/carbon/human/life.dm:1253]: Due to order of operations the plant statement is getting executed even when dna is null.  I already have it open so I can fix it right now if necessary.

The && will properly short-circuit the slime statement out, but it doesn't wipe out the whole line, just the next statement.  If it were a chain of ands, the zero would propagate and negate them all, but logical or doesn't work like that.

**Pull request merged**